agent/e2e/agent-tui/utils.sh

```diff
@@ -73,4 +81,33 @@ function pressKeys(){
     local keyCode="KEY_"$c
     _pressKey $keyCode
   done
-}
\ No newline at end of file
+}
+
+function typeIPAddress() {
```

Comment:
Rethinking, maybe this is not needed as the existing `pressKeys()` could be used

Reply:
I don't think it is needed.